pkg/template/helper.go

```diff
@@ -42,3 +44,35 @@ func (r TemplateReference) String() string {
 	}
 	return r.Name
 }
+
+// ConvertUserToTemplateInstanceRequester copies analogous fields from user.Info to TemplateInstanceRequester
+func ConvertUserToTemplateInstanceRequester(u user.Info) templateapi.TemplateInstanceRequester {
```

Comment:
this can all go away if we can make TemplateInstanceRequester just be of type DefaultInfo